pkg/asset/manifests/gcp/cluster.go

```diff
@@ -0,0 +1,150 @@
+package gcp
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/apparentlymart/go-cidr/cidr"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/ptr"
+	capg "sigs.k8s.io/cluster-api-provider-gcp/api/v1beta1"
+	"sigs.k8s.io/kustomize/kyaml/sets"
```

Comment:
Didn't you mean to use 
```suggestion
	"k8s.io/apimachinery/pkg/util/sets"
```